DataFormats/Common/interface/FwdRef.h

```diff
@@ -113,6 +113,8 @@ FwdRef: A template for a interproduct reference to a member of a product.
 #include "DataFormats/Common/interface/CMS_CLASS_VERSION.h"
 #include "DataFormats/Common/interface/Ref.h"
 
+#include <boost/functional.hpp>
```

Comment:
Why a new boost header is needed?

Reply:
The file has always implicitly depended upon that header. I just made it explicit.

Reply:
Thanks. The dependence comes from `boost::binary_traits`, right?

Reply:
Yes